### PR TITLE
[KIECLOUD-698] - Retrieve the image name built os OSBS for the operators

### PR DESCRIPTION
**Pull request by spolti**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[link](https://examle.com)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* referenced pull request 1
* referenced pull request 2
* referenced pull request 2
etc. 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/osbs/bamoe_operators_osbs_seed_cloud_jobs.groovy [hunk 1]
```diff
@@ -158,9 +158,25 @@ prodComponent.each { Component ->
                           checkout(githubscm.resolveRepository(operator_repo_name, GITHUB_ORG_UNIT, operator_branch, false))
                           // Run the build script that should be into the operator hack folder
                           dir('hack') {
-                            sh "source ~/virtenvs/cekit/bin/activate && ./${build_command}"
+                            sh "source ~/virtenvs/cekit/bin/activate && ./${build_command} | tee output.txt"
                           }
+                          
                           // post processing
+                          // query the built image from osbs using brew cli
+                          dir(component_path) {
```
Comment (lampajr) [inline job-dsls/jobs/osbs/bamoe_operators_osbs_seed_cloud_jobs.groovy:166]:
```suggestion
                          dir('hack') {
```

File: job-dsls/jobs/osbs/rhpam_operators_osbs_seed_cloud_jobs.groovy [hunk 2]
```diff
@@ -157,9 +157,24 @@ prodComponent.each { Component ->
                           checkout(githubscm.resolveRepository(operator_repo_name, GITHUB_ORG_UNIT, operator_branch, false))
                           // Run the build script that should be into the operator hack folder
                           dir('hack') {
-                            sh "source ~/virtenvs/cekit/bin/activate && ./${build_command}"
+                            sh "source ~/virtenvs/cekit/bin/activate && ./${build_command} | tee output.txt"
                           }
+
                           // post processing
+                          // query the built image from osbs using brew cli
+
```
Comment (lampajr) [inline job-dsls/jobs/osbs/rhpam_operators_osbs_seed_cloud_jobs.groovy:165]:
```suggestion
                          dir('hack') {
```

File: job-dsls/jobs/osbs/bamoe_operators_osbs_seed_cloud_jobs.groovy (hunk 1 above)
Comment (lampajr) [reply job-dsls/jobs/osbs/bamoe_operators_osbs_seed_cloud_jobs.groovy:166]: another option could be to define a new variable `component_path` and set it equal to `hack` and use it

File: job-dsls/jobs/osbs/rhpam_operators_osbs_seed_cloud_jobs.groovy (hunk 2 above)
Comment (lampajr) [reply job-dsls/jobs/osbs/rhpam_operators_osbs_seed_cloud_jobs.groovy:165]: As before the `output.txt` is created inside `hack` folder

File: job-dsls/jobs/osbs/bamoe_operators_osbs_seed_cloud_jobs.groovy (hunk 1 above)
Comment (spolti) [reply job-dsls/jobs/osbs/bamoe_operators_osbs_seed_cloud_jobs.groovy:166]: it was a leftover, imho we can stick with dir(hack)

**Review by davidesalerno — APPROVED** [review]

**Review by lampajr — APPROVED** [review]

**Pull request merged**